database/src/main/scala/cromwell/database/SqlDatabase.scala

```diff
@@ -8,144 +8,194 @@ import scala.concurrent.{ExecutionContext, Future}
                                       (implicit ec: ExecutionContext): Future[Traversable[WorkflowMetadataSummary]]
 
+  @deprecated("Olde Worlde Databasee Tablee", "0.21")
   def countWorkflowSummaries(statuses: Set[String], names: Set[String], uuids: Set[String],
                                        startDate: Option[Timestamp], endDate: Option[Timestamp])
                                       (implicit ec: ExecutionContext): Future[Int]
 
+  @deprecated("Olde Worlde Databasee Tablee", "0.21")
   def updateCallCaching(workflowUuid: String, allow: Boolean)(implicit ec: ExecutionContext): Future[Int]
 
+  @deprecated("Olde Worlde Databasee Tablee", "0.21")
   def updateCallCaching(workflowUuid: String, allow: Boolean, callFqn: String, attempt: Int)
                                  (implicit ec: ExecutionContext): Future[Int]
 
+  @deprecated("Olde Worlde Databasee Tablee", "0.21")
   def updateCallCaching(workflowUuid: String, allow: Boolean, callFqn: String, index: Int, attempt: Int)
                                  (implicit ec: ExecutionContext): Future[Int]
 
+  @deprecated("Olde Worlde Databasee Tablee", "0.21")
   def infosByExecution(workflowUuid: String)
                                 (implicit ec: ExecutionContext): Future[Traversable[(Execution, ExecutionInfo)]]
 
+  @deprecated("Olde Worlde Databasee Tablee", "0.21")
   def infosByExecution(workflowUuid: String, callFqn: String)
                                 (implicit ec: ExecutionContext): Future[Traversable[(Execution, ExecutionInfo)]]
 
+  @deprecated("Olde Worlde Databasee Tablee", "0.21")
   def callCacheDataByExecution(workflowUuid: String)(implicit ec: ExecutionContext):
   Future[Traversable[(Execution, Option[String], Option[String])]]
 
+  @deprecated("Olde Worlde Databasee Tablee", "0.21")
   def getExecutionsWithResuableResultsByHash(hash: String)
                                                       (implicit ec: ExecutionContext): Future[Traversable[Execution]]
 
+  @deprecated("Olde Worlde Databasee Tablee", "0.21")
   def runningExecutionsAndExecutionInfos(workflowUuid: String, statuses: Set[String])
-                                                  (implicit ec: ExecutionContext):
-  Future[Traversable[(Execution, ExecutionInfo)]]
+                                                  (implicit ec: ExecutionContext): Future[Traversable[(Execution, ExecutionInfo)]]
+
+
+  /*
+  The following section relates to:
+.___  ___.  _______ .___________.    ___       _______       ___   .___________.    ___
+|   \/   | |   ____||           |   /   \     |       \     /   \  |           |   /   \
+|  \  /  | |  |__   `---|  |----`  /  ^  \    |  .--.  |   /  ^  \ `---|  |----`  /  ^  \
+|  |\/|  | |   __|      |  |      /  /_\  \   |  |  |  |  /  /_\  \    |  |      /  /_\  \
+|  |  |  | |  |____     |  |     /  _____  \  |  '--'  | /  _____  \   |  |     /  _____  \
+|__|  |__| |_______|    |__|    /__/     \__\ |_______/ /__/     \__\  |__|    /__/     \__\
+   */
```

Comment:
This was vintage from before I moved the WorkflowStore stuff into a separate trait. Probably not over useful any more